source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -1238,7 +1238,7 @@ namespace RendererQml
 		auto month_Text = "getText(0,3)";
 		auto day_Text = "getText(4,6)";
 		auto year_Text = "getText(7,11)";
-		std::string DateRegex = Formatter() << "/^" << month_Regex << dateSeparator << day_Regex << dateSeparator << year_Regex << "$/";
+		std::string DateRegex = Formatter() << "/^" << month_Regex << "\\" << dateSeparator << day_Regex << "\\" << dateSeparator << year_Regex << "$/";
```

Comment:
can we not do `const auto dateSeparator = "\\/";` instead of adding everywhere?

Reply:
Yeah this works. Thanks for the suggestion.
I have added this change in the latest commit